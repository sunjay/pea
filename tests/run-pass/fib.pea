fn fib(n) {
    if (n < 2) {
        return n;
    }

    fib(n - 2) + fib(n - 1)
}

fn main() {
    println!(b"fib(35)");
    let value = fib(35);
    println!(value);
}
