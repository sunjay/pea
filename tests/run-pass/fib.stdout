fib(35)
9227465
